js/main.js

```diff
@@ -325,7 +324,7 @@
                 return;
             }
 
-            var navHtml = marked(navMD);
+            var navHtml = marked.parse(navMD);
```

Comment:
Since [marked release v4.0.0](https://github.com/markedjs/marked/releases/tag/v4.0.0) it's necessary to use marked.parse(string).